### PR TITLE
Fixing variable links leading to the ValidMind UI

### PR DESCRIPTION
**Pull request by noosheenv**
## Internal Notes for Reviewers

- In the style guide under Product names, I added a row to the table for the new variable, `{{< var vm_url >}}`.
I also added a Description column that describes each variable. 

| Before | After |
| ------ | ------ |
| image | image |

- In the same section, I changed the wording in the introductory text from `product features` -> `constants`.

| Before | After |
| ----- | ----- |
| image | image |

- I replaced references to `({{< var vm_ui >}})` to `({{< var vm_url >}})` when the link was intended to take us to the landing page of the ValidMind UI.

| Before | After |
| ----- | ----- |
| image | image |
| image | image |

- I replaced instances of `(https://app.prod.validmind.ai)` to `({{< var vm_url >}})` for good code style.

### CODE REVIEW

**Comment by validbeck** [comment]
- I'm going to pull this down in a second to review the files properly, but I wanted to make sure that at some point you called `make get-source` to retrieve the latest notebooks before rendering?
- Have a think about the definition of the `ValidMind AI risk platform` again as well, as what you have right now doesn't really tell readers what ValidMind does. ;) (Hint, have you read the **Glossary** and About **ValidMind** pages?)

**Comment by noosheenv** [comment]
Just pushed an update after running `make get-source` and `quarto render`, and a new definition after referring to the glossary and ChatGPT. Thanks for catching me!

**Review by validbeck — CHANGES_REQUESTED** [review]
Nicely done! Just a few changes, and something to consider for the future (I had hoped you would dig into it a little yourself out of curiosity, so I didn't include it in the explicit directions, but...) is **other** instances of `https://app.prod.validmind.ai` that could use the variable instead. ;) 

For example: 
- https://app.prod.validmind.ai/model-inventory > `{{< var vm_url >}}/model-inventory`
- And any other pages that exist in under the parent platform UI URL!
- There's also some instances of `<https://app.prod.validmind.ai/>` that should be replaced with `<{{< var vm_url >}}>` and so on.

That's OK — you fixed something that was a bug we introduced, and that's huge! It takes time and a few passes to catch everything, and even then it doesn't have to be perfect. Just trying to get you to start thinking about edge cases, as getting practice thinking in that mindset will really be beneficial. 

For the record, my search results (include: `.qmd`, exclude: `.html, .ipynb`): 

```
36 results - 18 files

site/guide/assess-compliance.qmd:
  17  
  18: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  19  

  44  
  45: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  46  

  68  
  69: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  70  

site/guide/export-documentation.qmd:
  18  
  19: 1. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  20  

  36  
  37: 1. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  38  

site/guide/get-started-sandbox.qmd:
   37  <!--- FUTURE step for self-signup
   38: 1. Get your access credentials for the {{< var vm_ui >}} by signing up with ValidMind: <https://app.prod.validmind.ai/>
   39     

   45     - ValidMind Jupyter Hub: <https://jupyterhub.validmind.ai/>
   46:    - {{< var vm_platform >}}: <https://app.prod.validmind.ai/>
   47  

  114  
  115: <form method="get" action="https://app.prod.validmind.ai/" target="_blank">
  116     <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 16px; padding: 6.25px 12.5px; margin-left: 16px; margin-bottom: 20px;">ValidMind Platform UI</button>

site/guide/install-and-initialize-developer-framework.qmd:
  26  
  27: 1. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  28  

site/guide/preparing-validation-reports.qmd:
  57  
  58: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  59  

site/guide/submit-for-approval.qmd:
  21  
  22: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  23  

  42  
  43: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  44  

site/guide/swap-documentation-templates.qmd:
  18  
  19: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  20  

site/guide/view-documentation-activity.qmd:
  19  
  20: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  21  

site/guide/view-documentation-guidelines.qmd:
  22  
  23: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  24  

site/guide/view-documentation-templates.qmd:
  19  
  20: 2. In the left sidebar, click **Settings** and then [**Templates**](https://app.prod.validmind.ai/settings/templates).
  21  

site/guide/view-reports.qmd:
  42  
  43: 2. In the left sidebar, click **[Reports](https://app.prod.validmind.ai/reports)**.
  44  

site/guide/view-validation-guidelines.qmd:
  20  
  21: 1. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  22  

site/guide/work-with-content-blocks.qmd:
  28  
  29: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  30  

  57  
  58: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  59  

site/guide/work-with-model-findings.qmd:
  21  
  22: 2. In the left sidebar, click [**Model Findings**](https://app.prod.validmind.ai/model-findings).
  23  

  41  
  42: 2. In the left sidebar, click [**Model Findings**](https://app.prod.validmind.ai/model-findings) and then click {{< fa filter >}} **Filter**.
  43  

  58  
  59: Filters can be removed by clicking on the {{< fa xmark >}} next to them on the main [**Model Findings**](https://app.prod.validmind.ai/model-findings) page where the results are displayed. 
  60  

  66  
  67: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  68  

  91  
  92: 2. In the left sidebar, click [**Model Findings**](https://app.prod.validmind.ai/model-findings).
  93  

site/guide/working-with-model-documentation.qmd:
  62  
  63: 2. In the left sidebar, click [**Model Inventory**](https://app.prod.validmind.ai/model-inventory).
  64  

site/releases/2024-jan-18/highlights.qmd:
  56  
  57: Try it in the {{< var vm_ui >}}: [User directory](https://app.prod.validmind.ai/settings/user-directory)
  58  

  68  
  69: Try it in the {{< var vm_ui >}}: [Groups](https://app.prod.validmind.ai/settings/groups)
  70  

  87  
  88: Try it in the ValidMind UI: [Templates](https://app.prod.validmind.ai/settings/templates)
  89  

site/releases/2024-mar-27/highlights.qmd:
  54  
  55: - [Model Findings](https://app.prod.validmind.ai/model-findings)
  56: - [Reports](https://app.prod.validmind.ai/reports)
  57  

  75  
  76: - [Model Inventory](https://app.prod.validmind.ai/model-inventory)
  77: - [Model Findings](https://app.prod.validmind.ai/model-findings)
  78  

  89  
  90: Try it: [Custom Fields](https://app.prod.validmind.ai/settings/inventory-model-custom-fields)
  91  

site-unused/get-started-closed-beta.qmd:
   52  <!--- TO DO Do we need the OW folks to follow this step? --->
   53: 1. Get your credentials for the {{< var vm_ui >}} by signing up with ValidMind: <https://app.prod.validmind.ai/>
   54     

  100  
  101: Jupyter notebooks will often tell you where to look specifically after you run the sample code, but the login for the {{< var vm_ui >}} is always the same: <https://app.prod.validmind.ai/> 
  102  
```

**Comment by validbeck** [comment]
> a new definition after referring to the glossary and ChatGPT. Thanks for catching me!

Looks better, but what about...

```
| {{< var vm_risk >}} | `{{{< var vm_risk >}}}` | Risk platform with a {{< var vm_dev >}} for documenting and testing models, alongside a {{< var vm_ui >}} hosting cloud-based tools, APIs, databases, and validation engines. |
```

;)

File: site/about/style-guide.qmd [hunk 1]
```diff
@@ -202,19 +202,20 @@ In the context of model risk management, proper nouns include specific models, l
...
 
-| Product Name | Variable Key |
-|------:|-----|
-| {{< var vm_risk >}} |  `{{{< var vm_risk >}}}` |
-| {{< var vm_framework >}} |  `{{{< var vm_framework >}}}` |
-| {{< var vm_platform >}} |  `{{{< var vm_platform >}}}` |
-| {{< var vm_dev >}} |  `{{{< var vm_dev >}}}` |
-| {{< var vm_ui >}} |  `{{{< var vm_ui >}}}` |
+| Product Name | Variable Key | Description |
+|----:|-----|:-------|
```
[the remark this answers is not in the document]
Comment (noosheenv) [reply site/about/style-guide.qmd:212]: What size window are we aiming for when it comes to fitting the text in one row? When I have it on my 11-inch iPad, neither implementation fits it in one row. On the 14-inch MacBook, with both implementations, I can resize the window to about 75% before it resizes the table and fits the text into multiple rows. Is there a standard size I should model for, or is there a rule of thumb/default to go with? Like should I normally stick with `|----|` and use the `:` sparingly?

**Comment by noosheenv** [comment]
> * `<https://app.prod.validmind.ai/>` that should be replaced with `<{{< var vm_url >}}>` and so on.

I tested that and it turned into not-a-link, so I did some testing and this works as expected: `[{{< var vm_url >}}]({{< var vm_url >}})`, a little clunky but it works!

File: site/about/style-guide.qmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (validbeck) [reply site/about/style-guide.qmd:212]: I've left a suggestion for the sizes--you can just commit it!

**Review by validbeck — APPROVED** [review]

**Comment by validbeck** [comment]
@noosheenv Amazing! You didn't have to even make another Story, well done. Looks good to go, go ahead and follow the `Merge into main` instructions on "GitHub guides".

**Pull request merged**